recipes/provenance/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "provenance" %}
+{% set version = "0.9.0" %}
+{% set sha256 = "50d5f09449a8cc96c8cbd6f49769f8bede3ab507efcbed446a967627163f14e4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python >= 3.5
```

Comment:
There is no indication from the code that this package should require python >=3.5. If you look at the following, it would seem like it supports python 2 and 3:

https://github.com/Savvysherpa/provenance/blob/master/provenance/compatibility.py